routers/index.js

```diff
@@ -1,16 +1,21 @@
 import { Router } from 'express'
-import { usersRouter } from './usersRouter.js'
 import { productsRouter } from './productsRouter.js'
+import { cartsRouter } from './cartsRouter.js'
 
 const router = Router()
 
 router.get('/', (req, res) => {
-  res.json({
-    message: 'this is test message for root endpoint',
-  })
+  const dogAscii = `  
+   __
+o-''|\\_____/)
+ \\_/|_)     )
+    \\  __  /
+    (_/ (_/ 
+`
+  res.send(`<pre>${dogAscii}</pre>`)
 })
 
-router.use('/users', usersRouter)
 router.use('/products', productsRouter)
+router.use('/shopping-cart', cartsRouter)
```

Comment:
```suggestion
router.use('/products', productsRouter)
router.use('/shopping-carts', cartsRouter)
```
사소하다고 생각될 수 있지만 복수, 단수도 일괄적으로 맞춰주시는게 좋습니다 :)
(ex. `/shopping-carts`, `/carts`)